### PR TITLE
A better upstart file

### PR DESCRIPTION
**Pull request by cmouse**
Avoids some IPv6 issues.

### CODE REVIEW

File: contrib/upstart-recursor.conf [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # recursor - the PowerDNS Recursor
 # for in /etc/init/recursor.conf
 description "PowerDNS Recursor"
+start on static-network-up
```
Comment (Habbie) [inline contrib/upstart-recursor.conf]: This does not work on RHEL6, I think

Comment (pieterlexis) [inline contrib/upstart-recursor.conf]: According to [this](http://upstart.ubuntu.com/cookbook/#standard-idioms) you could use `start on stopped network` in older versions. As the static-network-up refers to the interfaces listed in `/etc/network/interfaces`.

**Comment by Habbie** [comment]
Anybody feel like finishing this?

**Comment by pieterlexis** [comment]
Closing this for lack of activity and this breaks on RHEL 6 (th eonly real OS with upstart we still target)

**Pull request closed without merging**